lib/api/dsl/methods.js

```diff
@@ -7,8 +7,202 @@ var
   q = require('q'),
   util = require('util'),
   geohash = require('ngeohash'),
+  units = require('node-units'),
```

Comment:
This library has not been updated for 3 years, and it uses an even more outdated version of ``numerizer`` as a dependency.

[convert-units](https://www.npmjs.com/package/convert-units) seems more up-to-date.